example/20-network.yaml

```diff
@@ -46,11 +46,15 @@ spec:
   providerConfig:
     apiVersion: calico.networking.extensions.gardener.cloud/v1alpha1
     kind: NetworkConfig
-    backend: bird
-    ipam:
-      type: host-local
-      cidr: usePodCIDR
+    overlay:
+      enabled: false
+    snatToUpstreamDNS:
+      enabled: true
+#   ebpfDataplane:
+#     enabled: true
 #   ipAutodetectionMethod: interface=eth0
-#   ipip: CrossSubnet
 #   typha:
 #     enabled: true
+#   ipam:
+#     type: host-local
+#     cidr: usePodCIDR
```

Comment:
Nit: Any particular reason why you moved `ipam` to the bottom?

Reply:
I've never seen that anybody has set this field and do we really want stakeholder to set this?

Reply:
Ah, I did not get that you commented it. For sure, this is expert configuration, which should not necessarily be set. No issue to keep it commented out.